bedwars-plugin/src/main/java/com/andrei1058/bedwars/configuration/GeneratorsConfig.java

```diff
@@ -30,7 +30,7 @@
     public GeneratorsConfig(Plugin plugin, String name, String dir) {
         super(plugin, name, dir);
 
-        if (isFirstTime()) {
+        //if (isFirstTime()) {
```

Comment:
If you are going to remove this, please actually remove it, then fix the indenting for the lines below.